_posts/2018-01-19-basic-setup.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "basic setup and beginner tutorials"
+category: tutorial
+date: 2018-01-18 13:14:00
+---
+
+## Dependencies
+
+If you don't already have a set workflow for Haskell development, may I suggest
+
+- [stack](https://haskell-lang.org/get-started)
+
+    User-friendly tool to set up projects and run builds.
+
+- [emacs](https://www.gnu.org/software/emacs/)
```

Comment:
Probably emacs is a bit too hard for "everyone" (this means the 70% of people who dont use Vim/Emacs). I'd suggest VS Code, as it has some great editor integration using https://github.com/alanz/vscode-hie-server